src/helpers/validators/validateString/index.ts

```diff
@@ -0,0 +1,128 @@
+// Import helpers
+import validateRegex from '../shared/helpers/validateRegex';
+
+// Import types
+import ValidationResult from '../shared/types/ValidationResult';
+import StringValidationRequirements from './StringValidationRequirements';
+
+/*------------------------------------------------------------------------*/
+/* ------------------------------ Constants ----------------------------- */
+/*------------------------------------------------------------------------*/
+
+// define minimum and maximum range for lowercase ASCII chars
+const LOWERCASE_MIN = 65;
+const LOWERCASE_MAX = 90;
+
+// define minimum and maximum range for uppercase ASCII chars
+const UPPERCASE_MIN = 97;
+const UPPERCASE_MAX = 122;
+
+// define minimum and maximum range for ASCII digits
+const DIGIT_MIN = 48;
+const DIGIT_MAX = 57;
+
+/**
+ * Determines whether a given input string is considered valid based on
+ *   the provided requirements.
+ * @author Austen Money
+ * @param input input string
+ * @param reqs requirements that the provided string should conform to
+ * @returns whether input is considered valid according to reqs - if
+ *   valid, returns a cleaned version of input; if invalid, returns
+ *   a string containing error messages describing which requirements
+ *   were not met.
+ */
+const validateString = (
+  input: string,
+  reqs: StringValidationRequirements,
```

Comment:
Is `StringValidationRequirements` only used here? If so, we generally try to include argument types right here, either in this file or inline right here.

Reply:
I like having them inline there! It seems like it would make them easier to find and refer back to - just moved them.